.gitignore

```diff
@@ -16,3 +16,6 @@ serviceprincipal.json
 bin/
 obj/
 out/
+.DS_Store
+*.DS_Store
+opwen_email_server/config.py
```

Comment:
Does this need to be here?

Reply:
No sir! will be removed

Reply:
Although, it would be convenient for me to keep the .DS_Store. I can figure out a workaround if you'd like though.